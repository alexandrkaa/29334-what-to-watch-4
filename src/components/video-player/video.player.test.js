import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const movie = {
  id: 4,
  image: `img/we-need-to-talk-about-kevin.jpg`,
  movieBackground: `img/bg-the-grand-budapest-hotel.jpg`,
  movieDate: `2000`,
  movieDescription: [`In the 1930s, the Grand Budapest Hotel is a popula…y boy, becomes Gustave&apos;s friend and protege.`, `Gustave prides himself on providing first-classNam…ess painting and the chief suspect in her murder.`],
  movieDirector: `Wes Anderson`,
  movieGenre: `Thriller`,
  movieImage: `img/the-grand-budapest-hotel-poster.jpg`,
  movieRatingCount: `349 ratings`,
  movieRatingLevel: `Bad`,
  movieRatingScore: `3`,
  movieStarring: `Jude Law, Willem Dafoe, James Franco, Jason Statham, Tom Hardy, Saoirse Ronan, Tony Revoloru, Tilda Swinto`,
  title: `Dardjeeling Limited`,
  moviePreview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
};

it(`<VideoPlayer /> should render correctly`, () => {
  const isPlaying = true;
  const isMuted = true;
  const {moviePreview, image} = movie;
  const tree = renderer
    .create(
        <VideoPlayer
          isMuted={isMuted}
          isPlaying={isPlaying}
          src={moviePreview}
          poster={image}
        />,
        {
          createNodeMock: () => {
            return {};
          }
        }
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
