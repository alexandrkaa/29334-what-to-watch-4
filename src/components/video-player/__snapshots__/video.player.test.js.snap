// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer /> should render correctly 1`] = `
<video
  autoPlay={false}
  controls={false}
  height={175}
  preload="metadata"
  width={280}
/>
`;
